content/courses/mobile/intro-to-solana-mobile.md

```diff
@@ -132,297 +124,320 @@ Unreal Engine.
+
+> While traditional app stores impose strict policies around blockchain
+> transactions to protect their revenue and comply with regulations, alternative
+> distribution methods like the Solana dApp Store offers developers a more
+> flexible platform for deploying Solana-based mobile applications. This
+> decentralized approach bypasses many of the restrictions seen in centralized
+> app marketplaces, allowing dApps to thrive in a more blockchain-friendly
+> ecosystem.
+
+## Conclusion
+
+Getting started with Solana mobile development is more accessible than ever,
+thanks to the Solana Mobile Stack (SMS). Although React Native introduces some
+differences compared to React, much of the code you will write remains familiar,
+particularly when it comes to structuring the UI and handling state. The main
+distinction lies in how you interact with wallets, which requires using the
+`transact` callback to establish wallet sessions, sign transactions, and
+communicate with Solana’s blockchain.
+
+As you continue building Solana mobile dApps, it's essential to keep learning
+and refining your skills. Be sure to explore additional resources like:
+
+- [The official Solana Developer Docs](https://solana.com/docs) for in-depth
+  guides on Solana’s core libraries and best practices.
+- [Solana's Discord community](https://discord.com/invite/kBbATFA7PW) forum for
+  troubleshooting, sharing insights, and staying updated on the latest ecosystem
+  changes.
+
+Mastering mobile Solana development will open up new opportunities in
+decentralized finance (DeFi), gaming, and e-commerce, allowing you to build
+cutting-edge applications with a seamless user experience. Stay curious and
+experiment with different tools to push the boundaries of what you can achieve
+with mobile dApps. Let's put our knowledge to test by building a counting app
+with React Native for Android OS!
+
+## Lab: Building a Mobile Counter dApp with React Native
+
+This dApp will display a counter and allow users to make increments via a
+transaction on the Solana blockchain. The app will also connect to a wallet for
+signing transactions.
+
+We will use the **Anchor framework** to interact with the on-chain counter
+program. The client side has already been developed in one of our previous
+lessons called
+[Intro to client-side Anchor development](https://solana.com/developers/courses/onchain-development/intro-to-anchor-frontend),
+feel free to check out its code for more context.
+
+To ensure you fully understand the core concepts, we will write this application
+in vanilla React Native without a starting template. While Solana Mobile offers
+templates that handle some boilerplate, building from scratch provides a much
+deeper understanding.
+
+### Getting Started
+
+To get started, you will need to properly set up a React Native development
+environment if you didn't already. This
+[article](https://reactnative.dev/docs/set-up-your-environment) shows you how.
```

Comment:
```suggestion
To get started, you will need to properly set up a React Native development
environment if you didn't already. The
[React native Environment Setup Guide](https://reactnative.dev/docs/set-up-your-environment) shows you how.
```

Make link text useful without needing to read the surrounding text. 'article' on its own doesn't have enough detail. See [CONTRIBUTING.med](https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#prose)